tasks/configure.yml

```diff
@@ -0,0 +1,67 @@
+---
+# ss-configure
+
+# Create api-key if not defined
+- set_fact: archivematica_src_configure_ss_api_key={{ 999999999999999999999 | random | to_uuid | hash('md5') }}
+  when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"
+
+# Delete default SS superuser test
+- name: "Delete default SS superuser"
+  become: "yes"
+  shell: |
+    cd {{ archviematica_src_ss_app }}
+    export SS_DB_NAME={{ archivematica_src_ss_environment['SS_DB_NAME'] }}
```

Comment:
I think you can set the env in a line, as we do here: https://github.com/artefactual-labs/ansible-archivematica-src/blob/a38b2d1a8a972f51f395cffb6cf52ef2155b291d/tasks/ss-db.yml#L17